tests/bdd/features/ha/node-agent/test_path_replacement.py

```diff
@@ -125,7 +125,7 @@ def background():
         VOLUME_UUID, Protocol("nvmf"), node=TARGET_NODE_1
     )
     yield volume
-    Deployer.stop()
+    # Deployer.stop()
```

Comment:
guess this is from debug, we should add stop back.

Reply:
Oops, reverting it back